src/carma-platform/parse_ros_bags.py

```diff
@@ -59,7 +59,12 @@ def get_detected_objects(ros_bag_file, output_file, time_offset):
             ["Message Time (ms)", "Object ID", "Map Position X (m)", "Map Position Y (m)"]
         )
         for message in messages:
-            cdasim_time_ms = (message.header.stamp - time_offset).to_sec() * 1_000
+
+            try:
+                cdasim_time_ms = round((message.header.stamp - time_offset).to_sec() * 1_000)
+            except (TypeError):
+                cdasim_time_ms = 0
```

Comment:
If `cdasim_time_ms` is negative, should we skip this loop execution? I believe a negative time would indicate that CDASim is not up and running yet.